spec/lib/twingly/url_spec.rb

```diff
@@ -63,6 +60,21 @@ def valid_urls
     "http://:@blog.twingly.com/",
     "https://www.foo.ایران.ir/bar",
     "https://www.foo.xn--mgba3a4f16a.ir/bar",
+
+    # The following URL triggers
+    #
+    #   IDN::Idna::IdnaError: Output would be too large or too small (5)
+    #
+    # when used with Addressable and libidn (IDNA 2003)
+    # but considered valid with the pure Ruby IDNA solution in Addressable.
+    #
+    # This inconsistency has as been reported at
+    # https://github.com/sporkmonger/addressable/issues/239
+    #
+    # Long names like this will probably not resolv, see comments at
+    # https://github.com/twingly/twingly-url/issues/66#issuecomment-246349029
+    # but for now as addressable considers them valid, we do the same.
+    "http://AcinusFallumTrompetumNullunCreditumVisumEstAtCuadLongumEtCefallumEst.com",
```

Comment:
> This can jump back up again once Addressable 2.5.0 is released with sporkmonger/addressable@c73810f

Nope, I got that wrong too. With Addressable 2.5.0 and libidn, the URL is now considered valid.